full_bubble_model.py

```diff
@@ -328,7 +345,7 @@ def _pressure(t, R, R_dot, mu_L, surfactant, rho_L, p, p_dot, P_amb, args):
 # returns molar heat capacities, enthalpies and entropies
 @njit(float64[:, :](float64))
 def _thermodynamic(T):
-    ret = np.zeros((4, par.K), dtype=np.float64)   # [C_p, H, S, C_v]
+    ret = np.empty((4, par.K), dtype=np.float64)   # [C_p, H, S, C_v]
```

Comment:
`np.zeros` was changed to `np.empty`, where defaults values are not relevant. This improves array allocation times.